### PR TITLE
refactored classnames for sidebar

### PR DESCRIPTION
**Pull request by sumukhswamy**
Changed all the classnames for the sidebar component to avoid conflicts

### Check List
- [x] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by kavithacm** [comment]
looks like the tests are failing. Can you take a look into it? @sumukhswamy

**Review by mengweieric — APPROVED** [review]
LGTM thanks

File: public/components/event_analytics/explorer/sidebar/sidebar.tsx [hunk 1]
```diff
@@ -141,7 +141,7 @@ export const Sidebar = (props: ISidebarProps) => {
                   paddingSize="xs"
                 >
                   <ul
-                    className="dscSidebarList dscFieldList--selected"
+                    className="dscSidebarList explorerFieldList--selected"
```
Comment (mengweieric) [inline public/components/event_analytics/explorer/sidebar/sidebar.tsx:144]: minor: could we also rename 'dscSidebarList' with similar pattern?

File: public/components/event_analytics/explorer/sidebar/sidebar.tsx [hunk 2]
```diff
@@ -185,7 +185,7 @@ export const Sidebar = (props: ISidebarProps) => {
                 paddingSize="xs"
               >
                 <ul
-                  className="dscSidebarList dscFieldList--selected"
+                  className="dscSidebarList explorerFieldList--selected"
```
Comment (kavithacm) [inline public/components/event_analytics/explorer/sidebar/sidebar.tsx:188]: Same comment as Eric, could you rename this dscSidebarList as well?

**Review by kavithacm — COMMENTED** [review]
minor changes

**Review by kavithacm — APPROVED** [review]

**Pull request merged**